client/src/app/TabsProvider.js

```diff
@@ -219,7 +216,7 @@ export default class TabsProvider {
           } ];
         },
         getLinter() {
-          return BpmnLinter;
+          return null;
```

Comment:
This is a change in behavior. Is this intentional?

Reply:
The linter for Camunda 7 was a dummy that would never report any errors. Once we add rules for Camunda 7 the linter would be imported from `@camunda/linting` instead of the Camunda Modeler project.

Reply:
Done. 👍🏻